vanilla-tests/src/main/java/fixtures/url/multi/Queries.java

```diff
@@ -50,7 +49,7 @@
         @UnexpectedResponseExceptionType(ErrorException.class)
         Mono<Response<Void>> arrayStringMultiNull(
                 @HostParam("$host") String host,
-                @QueryParam("arrayQuery") String arrayQuery,
+                @QueryParam(value = "arrayQuery", multipleQueryParams = true) List<String> arrayQuery,
```

Comment:
I didn't find `multipleQueryParams` in azure-core?

Reply:
It's actually tied to the other PR, here:
[azure-sdk-for-java/pull/21203](https://github.com/Azure/azure-sdk-for-java/pull/21203)

Reply:
I see, yes, would be a useful feature.

This PR on autorest.java would need to wait for the one on azure-core and its release.

Reply:
It appears that `multipleQueryParams` is released with azure-core 1.19.0. And I've updated with it in latest v4 branch. If you want to continue this PR, you can try refresh latest v4 (there might be conflict to resolve as we've added quite a lot of code in meantime), and it might solve your CI.